main.go

```diff
@@ -330,6 +338,7 @@ func main() {
 	logger.Infow("restarting chaos-controller")
 
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
+		<-stopCh
```

Comment:
Why do we need to read from this channel (I'd expect to write on it if we want to stop the informer but not reading from it)? Also, should we write to the channel on exit?

Reply:
whoops, wrote this backwards